### PR TITLE
Set up logging using coloredlogs

### PR DESCRIPTION
**Pull request by rakaar**
Used [coloredlogs](https://coloredlogs.readthedocs.io/en/latest/readme.html) for colored logging. Debug messages printed in the `logs/debug.log` ([ref](https://www.youtube.com/watch?v=jxmzY9soFXg)) and info messages in `sys.stdout` ([ref](https://stackoverflow.com/questions/14058453/making-python-loggers-output-all-messages-to-stdout-in-addition-to-log-file))
fixes #[7]

### CODE REVIEW

File: app/test/colored_logging.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+''' Logging info, debug with coloredlogs '''
```
[the remark this answers is not in the document]
Comment (rakaar) [reply app/test/colored_logging.py]: actually the i tested the code this way: tried those commands in the python interpreter and checked if it was giving colored output or not. 
Where do I keep the file and should the name of the be same?

[the remark this answers is not in the document]
Comment (thealphadollar) [reply app/test/colored_logging.py]: Keep the file under `main/logging_config.py`

[the remark this answers is not in the document]
Comment (thealphadollar) [reply app/test/colored_logging.py]: Please update the structure in the README.md accordingly.

File: app/test/colored_logging.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+coloredlogs.install()
+
+DEBUG_LOGGER = logging.getLogger(__name__)
+INFO_LOGGER = logging.getLogger(__name__)
+
+DEBUG_LOGGER.setLevel(logging.DEBUG)
+INFO_LOGGER.setLevel(logging.INFO)
+
+FORMATTER = logging.Formatter(
+    '%(asctime)s %(hostname)s %(name)s[%(process)d] %(levelname)s %(message)s')
```
Comment (thealphadollar) [inline app/test/colored_logging.py]: We also need line number

Comment (rakaar) [reply app/test/colored_logging.py]: Included the line number with file name

File: app/main/logging_config.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+''' Logging info, debug with coloredlogs '''
```
Comment (thealphadollar) [inline app/main/logging_config.py]: I've a doubt? How are we importing this in Flask and using the settings there?

Can we use this inside other files? If yes, how?

Comment (rakaar) [reply app/main/logging_config.py]: importing the file just as module should do right ?

Comment (thealphadollar) [reply app/main/logging_config.py]: Please setup a small logging instance, like in manage.py with run command log ,"starting server on port xxxx"

**Comment by rakaar** [comment]
`logging_config.py` is set up in `app/main`. References used are [stackOverflow ans](https://stackoverflow.com/questions/46466408/flask-properly-configure-logging), [gist](https://gist.github.com/jakubczaplicki/c9077a482c2b195b7f3bcbee19dde441), [dict Schema from Docs](https://docs.python.org/2/library/logging.config.html#logging-config-dictschema).
**NOTE:**
We can use this file in flask just by importing, please refer  [this](https://github.com/geoHeil/flask_logging_question/blob/master/api.py).  which uses(imports from) [this file](https://github.com/geoHeil/flask_logging_question/blob/master/settings/settings.py)

**Pull request merged**